app/lib/routes/main_navigator/app_route.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:app/screens/home/home.dart';
+import 'package:app/screens/new_travel/new_travel_journey/test_screen_one.dart';
+import 'package:app/screens/new_travel/new_travel_journey/test_screen_two.dart';
+
+import 'package:flutter/material.dart';
+
+class AppRouteGenerator {
+  static Route<dynamic> onGenerateRoute(RouteSettings settings) {
```

Comment:
AppRouteGenerator contains our top level navigator. Though the 2 test screens are part of the inner navigation, it's still needed here to give us an option whether bottom nav will persist or not.